vic/drivers/shared_all/src/initialize_veg.c

```diff
@@ -39,35 +39,44 @@ initialize_veg(veg_var_struct **veg_var,
 
     for (i = 0; i < Nveg; i++) {
         for (j = 0; j < options.SNOW_BAND; j++) {
+            // Prognostic states
             veg_var[i][j].albedo = 0.0;
             veg_var[i][j].displacement = 0.0;
             veg_var[i][j].fcanopy = 0.0;
             veg_var[i][j].LAI = 0.0;
             veg_var[i][j].roughness = 0.0;
-            veg_var[i][j].throughfall = 0.0;
             veg_var[i][j].Wdew = 0.0;
             veg_var[i][j].Wdmax = 0.0;
```

Comment:
I agree that `Wdmax` isn't really a state or flux variable, it is really a derived model parameter. my suggestion would be to open a new issue that rethinks the handling of `Wdmax` and we can tackle that in another PR sometime down the road.